babel.config.js

```diff
@@ -8,9 +8,7 @@ const isBrowser = isCalypsoClient || 'true' === process.env.TARGET_BROWSER;
 const modules = isBrowser ? false : 'commonjs'; // Use commonjs for Node
 const codeSplit = require( './server/config' ).isEnabled( 'code-splitting' );
 
-const targets = isBrowser
-	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
-	: { node: 'current' };
+const targets = isBrowser ? undefined : { node: 'current' };
```

Comment:
The `undefined` value means "read the browserlists config from a supported location", right? Worth a comment that explains it.

Reply:
Good point, added!